app/controllers/general.php

```diff
@@ -563,25 +563,27 @@ function router(App $utopia, Database $dbForConsole, SwooleRequest $swooleReques
             throw new AppwriteException(AppwriteException::GENERAL_UNAUTHORIZED_SCOPE, $user->getAttribute('email', 'User') . ' (role: ' . \strtolower($roles[$role]['label']) . ') missing scope (' . $scope . ')');
         }
 
-        if (false === $user->getAttribute('status')) { // Account is blocked
-            throw new AppwriteException(AppwriteException::USER_BLOCKED);
-        }
-
-        if ($user->getAttribute('reset')) {
-            throw new AppwriteException(AppwriteException::USER_PASSWORD_RESET_REQUIRED);
-        }
+        if (in_array('api', $route->getGroups())) {
```

Comment:
Why do this if instead of just adding the block to an init hook we have that already runs only for this group?